src/screens/Quiz/components/Lives.tsx

```diff
@@ -1,54 +1,103 @@
 import { useGameContext } from "hooks";
 import { useEffect, useRef } from "react";
-import gsap from "gsap";
+import { gsap, Cubic } from "gsap";
 import styles from "./Lives.module.scss";
 
 const { life, lifeLost } = styles;
 
 export const Lives = () => {
   const { remainingLives, totalLives } = useGameContext();
-  const livesContainerRef = useRef(null);
+  const droppingLivesContainerRef = useRef(null);
 
   useEffect(() => {
     const ctx = gsap.context(() => {
       if (remainingLives === 2) {
-        gsap.to(`[data-effect=life-2]`, { fill: lifeLost, duration: 0.3 });
+        gsap.to(`[data-effect=droppingLife-2]`, {
```

Comment:
Don't use template literals **`** unless you are inserting some dynamic string value